src/styles/reset.scss

```diff
@@ -1,14 +1,8 @@
-html {
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  font-size: 16px;
-}
-
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
+.base-css-reset {
+  box-sizing: border-box;
   margin: 0;
-  font-weight: 600;
+  padding: 0;
+  line-height: normal;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 400;
 }
```

Comment:
if most of the components have their own .module.scss file with a `.componentName` class, why not set specfic reset values there instead of applying the same across the board?